src/RestWrite.js

```diff
@@ -1482,20 +1482,25 @@ RestWrite.prototype.objectId = function() {
 };
 
 // Returns a copy of the data and delete bad keys (_auth_data, _hashed_password...)
-RestWrite.prototype.sanitizedData = function() {
+RestWrite.prototype.sanitizedData = function(decodeData = true) {
```

Comment:
I don’t recommend passing default paramètres as truths, as different implementations with bottom values will produce different results (undefined, null,false)

Reply:
i will set it to false by default, but didn't know if its being used somewhere else. Was afraid from breaking something. Tests will show anyhow.